### PR TITLE
docs(community): Add page to gardening for a Spinnaker Summit Gardening Days workshop

### PR DESCRIPTION
**Pull request by prophen**
I added a page to the Spinnaker Gardening Days section, sponsor images, and a sidebar link to the new page. 

### Why is this change needed?

We are hosting a workshop in Gardening Days and we needed a place to conduct public voting on the participating teams' final projects.

### CODE REVIEW

File: community/gardening/spinnaker-product-box.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: single
+title: "Spinnaker Product Box Design Workshop"
+sidebar:
+  nav: community
+---
+## Presented by Capital Carbon Consulting
+> A "Product Box" is one of the experiential learning activities which can help the team identify the product values. We assume that the product will be packaged in a box for delivery to the customer and we will design this box together
```
Comment (317brian) [inline community/gardening/spinnaker-product-box.md]:
```suggestion
> A "Product Box" is one of the experiential learning activities that can help a team identify product values. We assume that the product will be packaged in a box for delivery to the customer, and we will design this box together.

File: community/gardening/spinnaker-product-box.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+- This is a 2.5-hour challenge to simulate a Product Box for Spinnaker
+- A DevOps Coach will guide this gamification challenge with step wise methodology, bringing the DevOps way to products
+- Best for Product Managers, Product Teams including UX Designers, Architects, Dev-Ops, Engineers, Cross functional teams
```
Comment (317brian) [inline community/gardening/spinnaker-product-box.md]: some nits:
- Product Managers are part of Product Teams
- `Cross functional teams` -> `cross-functional teams`

**Review by 317brian — COMMENTED** [review]
Some minor things

File: community/gardening/spinnaker-product-box.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+title: "Spinnaker Product Box Design Workshop"
+sidebar:
+  nav: community
+---
+## Presented by Capital Carbon Consulting
+> A "Product Box" is one of the experiential learning activities that can help a team identify product values. We assume that the product will be packaged in a box for delivery to the customer, and we will design this box together.
+
+- This is a 2.5-hour challenge to simulate a Product Box for Spinnaker
+- A DevOps Coach will guide this gamification challenge with step wise methodology, bringing the DevOps way to products
+- Best for Product Managers, Product Teams including UX Designers, Architects, Dev-Ops, Engineers, Cross functional teams
```
Comment (prophen) [inline community/gardening/spinnaker-product-box.md]:
```suggestion
- Best for product teams including product managers, UX designers, architects, DevOps, engineers, and cross-functional teams
```

File: community/gardening/spinnaker-product-box.md (hunk 2 above)
Comment (prophen) [reply community/gardening/spinnaker-product-box.md]: I made a change to implement your suggestion @brian-armory. What do you think?

Comment (317brian) [reply community/gardening/spinnaker-product-box.md]: 👍 I'll approve and merge once the tests finish

**Review by 317brian — APPROVED** [review]
LGTM

**Pull request merged**